beatrix/src/test/java/org/killbill/billing/beatrix/integration/overdue/TestOverdueIntegration.java

```diff
@@ -1142,7 +1142,7 @@ public void testOverdueClearWithCredit() throws Exception {
 
         final BigDecimal accountBalance = invoiceUserApi.getAccountBalance(account.getId(), callContext);
 
-        busHandler.pushExpectedEvents(NextEvent.INVOICE, NextEvent.BLOCK);
+        busHandler.pushExpectedEvents(NextEvent.INVOICE, NextEvent.INVOICE_ADJUSTMENT, NextEvent.INVOICE_ADJUSTMENT, NextEvent.BLOCK);
```

Comment:
Why two `INVOICE_ADJUSTMENT` events (might be worth adding a comment for the future reader...)?

Reply:
See cb5ae6d104cf